lib/importer/views/plugin_config.html

```diff
@@ -93,7 +93,40 @@ <h2 class="govuk-heading-l">Column settings</h2>
       </div>
 
     </div>
-  </div>
+      <form action="config" method="post">
+
+    <div class="govuk-grid-column-full govuk-!-padding-top-6" >
+        <h2 class="govuk-heading-l"></h2>
+
+<div class="govuk-form-group">
+  <fieldset class="govuk-fieldset" aria-describedby="waste-hint">
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+      <h1 class="govuk-fieldset__heading">
+        Report usage data
+      </h1>
+    </legend>
+
+    <p class="govuk-body">
+      Allow Register Dynamics to collect usage data on your use of the Data Upload Design Kit.
+      When enabled, the following events will be reported with no further data:
+      <ul class="govuk-list govuk-list--bullet">
+       <li>The Plugin was initialised</li>
+       <li>The Plugin was configured</li>
+      </ul>
+    </p>
+
+    <div class="govuk-checkboxes" data-module="govuk-checkboxes">
+      <div class="govuk-checkboxes__item">
+        <input class="govuk-checkboxes__input" id="collectData" name="collectData" type="checkbox" value="true" {% if collectUsageData == true %}checked{% endif %}>
+        <label class="govuk-label govuk-checkboxes__label" for="collectData">
+          Allow collection of usage data
+        </label>
+      </div>
+    </div>
+  </fieldset>
```

Comment:
Could we add an alternative call to action, eg "If you don't want usage data to be collected, please consider [dropping a note to us on Github](https://github.com/register-dynamics/data-import/discussions) to let us know how you're using the Kit. Thanks!"

Reply:
Added to 97ef5d9